scripts/ci/setup-dev-sso.sh

```diff
@@ -51,6 +51,22 @@ check_command()
     exit 1
 }
 
+run_wait_until_is_installed() {
+    WAIT_UNTIL_IS_INSTALLED=scripts/ci/wait-until-is-installed.sh
+    PARAMS="-crd keycloak.org -cs '' -n ${DEV_SSO_NS} -s ${SUBSCRIPTION_NAME}"
+
+    if [[ -f ${WAIT_UNTIL_IS_INSTALLED} ]]; then
+        source ${WAIT_UNTIL_IS_INSTALLED}
```

Comment:
I think that there is a bug also in the other script/file - it should be
```suggestion
        ${WAIT_UNTIL_IS_INSTALLED} ${PARAMS}
```

Reply:
thanks!